bundles/org.openhab.binding.doorbird/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
-	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
```

Comment:
What is the correct formatting here? IIRC correctly it only worked for me without code-style warnings if everything is on a single line.

Reply:
I don't see any warnings when using Maven or Eclipse. Maybe you use another IDE with a different opinion? XML parsers will ignore white spaces/new lines so it probably doesn't matter that much. I'm not aware of any convention we use for this so I would prefer to use the outcome of the Eclipse XML formatter everywhere so it doesn't result in diffs when someone uses it.